dist/json/platform-tokens.json

```diff
@@ -9537,7 +9537,7 @@
           "example": "text"
         },
         "original": {
-          "value": "{options.font.spacing.tight.value}",
+          "value": "{options.font.spacing.normal.value}",
```

Comment:
I wonder if we still have a working definition/use-case of font.spacing.tight at this point?